app/views/events/_filters.html.erb

```diff
@@ -1,119 +1,55 @@
 <button class="event-filters__toggle-button">
   Filter Events
-  <svg width="12px" height="8px" viewBox="0 0 12 8" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+  <svg class="toggle-arrow" width="12px" height="8px" viewBox="0 0 12 8" version="1.1" xmlns="http://www.w3.org/2000/svg">
     <title>Toggle Arrow</title>
     <desc>Toggles the filter menu</desc>
-    <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
-      <g transform="translate(-255.000000, -260.000000)" stroke="#F8E81C">
-        <g id="toggle-arrow" transform="translate(5.000000, 255.000000)">
-          <polyline points="251.199219 5.95703125 256.273391 12.0312033 260.80343 5.50116394"></polyline>
-        </g>
-      </g>
+    <g>
+      <polyline points="1.199219 0.95703125 6.273391 7.0312033 10.80343 0.50116394"></polyline>
     </g>
   </svg>
 </button>
 <div id="event-filters__filters" class="event-filters__filters">
-  <div class="event-filters__heading">Filter Events by:</div>
-
-  <div class="event-filters__group" role="group" aria-labelledby="filter-group-heading-topics">
+​
+  <div class="event-filters__group" data-filter-category="topics" role="group" aria-labelledby="filter-group-heading-topics">
     <h1 class="event-filters__group-heading" id="filter-group-heading-topics">Topics:</h1>
+    <% @topics.each do |topic| %>
```

Comment:
dynamically adding the filters based of the tags in use by events (stored in db)